includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```diff
@@ -3331,4 +3334,32 @@ public function is_payment_request_enabled() {
 		return in_array( WC_Stripe_Payment_Methods::APPLE_PAY, $enabled_payment_method_ids, true ) ||
 			in_array( WC_Stripe_Payment_Methods::GOOGLE_PAY, $enabled_payment_method_ids, true );
 	}
+
+	/**
+	 * Maybe deactivate Affirm or Klarna payment methods if other official plugins are active.
+	 *
+	 * @return void
+	 */
+	private function maybe_deactivate_bnpls() {
+		if ( WC_Stripe_Helper::has_other_bnpl_plugins_active() ) {
```

Comment:
Non-blocking: why not flip this into an early return?
```suggestion
		if ( ! WC_Stripe_Helper::has_other_bnpl_plugins_active() ) {
			return;
		}
```